tests/API/Management/BlacklistsTest.php

```diff
@@ -6,6 +6,8 @@
 
 class BlacklistsTest extends ApiTests {
 
+    private $domain;
```

Comment:
How come no `_` for this private property, or tests out of scope in this refactor?

Reply:
@cocojoe - I ran the `phpcs/phpcbf` against all the files and the list was so long that my terminal couldn't show it all 😆 so I just used it on `src/API/Authentication.php` (inspected the rest) to see how much work as involved. Answer is: a lot but I think it's worth it. I would like to see 5.x.x "finished" before it's archived and I see this as part of that.